pwndbg/heap/ptmalloc.py

```diff
@@ -658,6 +658,22 @@ def __init__(self) -> None:
         # ptmalloc cache for current thread
         self._thread_cache: gdb.Value = None
 
+    def largebin_size_range_from_index(self, index):
+        index += NSMALLBINS
+        spaces_table = self._spaces_table()
+        largest_largebin = self.largebin_index(pwndbg.gdblib.arch.ptrmask)
+        start_size = (NSMALLBINS * self.malloc_alignment) - self.malloc_alignment
+
+        for i in range(NSMALLBINS, index + 1):
+            start_size += spaces_table[i]
+
+        end_size = start_size + spaces_table[index + 1] - self.malloc_alignment
```

Comment:
this could be in an `else` of the if below; or:
```py
if index != largest_largebin:
    end_size = start_size + spaces_table[index + 1] - self.malloc_alignment
else:
    end_size = pwndbg.gdblib.arch.ptrmask
```

Reply:
Ah thank you for pointing this out 🙏